### ISSUE TITLE
Optimise console output

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#60]
It'd be cool if the console output could be optimized a bit. For instance given the following line:

`ComplexCondition - 5/3 - [timesFor] - Line/Column=(44,11) - Path=main/java/io/sunnyspots/android/io/model/Time.kt`

it would be beneficial to change a few things:

- include the project dir in the file (in my case `app/src/`)
- remove the `Path=` so that the file will be clickable in iTerm2 for instance
- Make use of the line number and column

I'd propose to change it to the following

```diff
-ComplexCondition - 5/3 - [timesFor] - Line/Column=(44,11) - Path=main/java/io/sunnyspots/android/io/model/Time.kt
+ComplexCondition - 5/3 - [timesFor] at app/src/main/java/io/sunnyspots/android/io/model/Time.kt:44:11
```

**Comment by arturbosch** [comment]
The input parameter in build.gradle is the project path which should be known to the user. I do not see the benefit showing it again and again. When you specify input as your git repository path, then detekt should show app/src.... for you. 

If I run detekt on detekt all sub modules are included in the path variable.

**Comment by vanniktech** [comment]
Yup doing `input = "${rootProject.file('.')}"` does the job. Thanks for the other optimizations.

**Comment by vanniktech** [comment]
Can we maybe revisit this? I've got `input = "${subProject.file('.')}"` now and when I execute `detektCheck` from the root directory I end up having `src/androidTest/java/com/vanniktech/chessclock/screenshots/ScreenshotsTest.kt` in the terminal where I can't click onto since the `app/` is missing.

**Comment by arturbosch** [comment]
Clicking on a link in the terminal seems like a good use case. Personally I like just having the package + file name to save space. 

A good idea would be maybe to provide terminal output customization next to output formatters like mentioned [here](https://github.com/arturbosch/detekt/issues/134). Feel free to open an issue for that ^^

**Comment by vanniktech** [comment]
Created #171
